arbitrator/vid-helper/src/bytes.rs

```diff
@@ -0,0 +1,160 @@
+//! Convenient serialization for binary blobs.
```

Comment:
Can use https://github.com/EspressoSystems/base64-bytes now

Reply:
This crate seems not have the full functionalities required here